frontend/src/breeding_list.tsx

```diff
@@ -123,6 +123,7 @@ export function BreedingList({ id }: { id: string | undefined }) {
             setActive(row);
           }}
           rowsPerPage={10}
+          orderBy="Födelsedatum"
```

Comment:
this needs to be `"birth_date"` instead of `"Födelsedatum"`.
Then it works.

Reply:
ah, nice. I Dont know how it was working for me then.